### ISSUE TITLE
PHP Warning: Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by growdev** [woothemes/woocommerce#10942]
## EXPLANATION OF THE ISSUE

PHP Warning:
[18-May-2016 19:31:58 UTC] PHP Warning:  Invalid argument supplied for foreach() in /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/class-wc-countries.php on line 234

If XDebug is turned on the Base Location is not usable. If in normal processing mode the page loads fine, but the warning is produced.

## STEPS TO REPRODUCE THE ISSUE

1. On a fresh install of WordPress 4.5.2, install WooCommerce 2.6.0 Beta 2
2. Activate WooCommerce and complete the setup wizard
3. In WordPress admin go to WooCommerce > Settings
4. The following warning is issued:

```
[18-May-2016 19:23:28 UTC] PHP Stack trace:
[18-May-2016 19:23:28 UTC] PHP   1. {main}() /Users/daniel/sites/wc26/wp-admin/admin.php:0
[18-May-2016 19:23:28 UTC] PHP   2. do_action() /Users/daniel/sites/wc26/wp-admin/admin.php:236
[18-May-2016 19:23:28 UTC] PHP   3. call_user_func_array:{/Users/daniel/sites/wc26/wp-includes/plugin.php:525}() /Users/daniel/sites/wc26/wp-includes/plugin.php:525
[18-May-2016 19:23:28 UTC] PHP   4. WC_Admin_Menus->settings_page() /Users/daniel/sites/wc26/wp-includes/plugin.php:525
[18-May-2016 19:23:28 UTC] PHP   5. WC_Admin_Settings::output() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/admin/class-wc-admin-menus.php:206
[18-May-2016 19:23:28 UTC] PHP   6. include() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/admin/class-wc-admin-settings.php:169
[18-May-2016 19:23:28 UTC] PHP   7. do_action() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/admin/views/html-admin-settings.php:26
[18-May-2016 19:23:28 UTC] PHP   8. call_user_func_array:{/Users/daniel/sites/wc26/wp-includes/plugin.php:525}() /Users/daniel/sites/wc26/wp-includes/plugin.php:525
[18-May-2016 19:23:28 UTC] PHP   9. WC_Settings_Page->output() /Users/daniel/sites/wc26/wp-includes/plugin.php:525
[18-May-2016 19:23:28 UTC] PHP  10. WC_Admin_Settings::output_fields() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php:102
[18-May-2016 19:23:28 UTC] PHP  11. WC_Countries->country_dropdown_options() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/admin/class-wc-admin-settings.php:573
[18-May-2016 19:23:28 UTC] PHP  12. WC_Countries->get_states() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/class-wc-countries.php:371
[18-May-2016 19:23:28 UTC] PHP  13. WC_Countries->load_country_states() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/class-wc-countries.php:137
[18-May-2016 19:23:28 UTC] PHP  14. WC_Countries->get_shipping_countries() /Users/daniel/sites/wc26/wp-content/plugins/woocommerce/includes/class-wc-countries.php:117
```

## SYSTEM STATUS REPORT

```
`
### WordPress Environment ###

Home URL: http://wc26.co
Site URL: http://wc26.co
WC Version: 2.6.0
Log Directory Writable: ✔
WP Version: 4.5.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: 5.6.10
PHP Post Max Size: 32 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
CURL Version: 7.43.0
OpenSSL/0.9.8zd

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.42 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ❌ wp_remote_post() failed. PayPal IPN won't work with your server. Contact your hosting provider. Error: Unsupported SSL protocol version
Remote Get: ✔

### Database ###

WC Database Version: 2.6.0
: 
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
PHP Post Max Size: ❌ The MaxMind GeoIP Database does not exist - Geolocation will not function. You can download and install it manually from http://dev.maxmind.com/geoip/legacy/geolite/ to the path:

### Active Plugins (1) ###

WooCommerce: by WooThemes – 2.6.0-beta-2

### Settings ###

Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔
API Version: 3.1.0

### WC Pages ###

Shop Base: #4 - /shop/
Cart: #5 - /cart/
Checkout: #6 - /checkout/
My Account: #7 - /my-account/

### Taxonomies ###

Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)


### Theme ###

Name: Twenty Sixteen
Version: 1.2
Author URL: https://wordpress.org/
Child Theme: ❌ – If you're modifying WooCommerce on a parent theme you didn't build personally
then we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –
`
```

**Comment by claudiosmweb** [comment]
@growdev already fixed in #10917 ;)

**Comment by growdev** [comment]
Ah, my search missed that. Thanks!
